src/expr.rs

```diff
@@ -1591,7 +1624,8 @@ pub fn rewrite_assign_rhs<S: Into<String>>(context: &RewriteContext,
                                            lhs: S,
                                            ex: &ast::Expr,
                                            width: usize,
-                                           offset: Indent)
+                                           offset: Indent,
+                                           try_block: bool)
```

Comment:
Could you use a new enum instead of a bool here please? It makes the caller code easier to read.